### PR TITLE
Update the quantities  homemade_tomato_sauce

### LINKED ISSUE #50: Homemade Tomato Sauce quantity
Tomato Sauce

### PR DESCRIPTION
**Pull request by Ali-Alshawabkeh**
fixes #50

### CODE REVIEW

File: homemade_tomato_sauce [hunk 1]
```diff
@@ -2,8 +2,17 @@
...
+2. 2 spones Olive oil
+3. 1 teaspon Salt
 
-## Instructions  
+## Instructions 
+1. Prepare Tomatoes: Blanch the tomatoes in boiling water for 1-2 minutes, then transfer to ice water. Peel, core, and chop them.
+
+2. Sauté Aromatics: In a large pan, heat olive oil over medium heat. Add the chopped onion and sauté until translucent. Add minced garlic and cook for another minute.
+
+3. Add Tomatoes: Stir in the chopped tomatoes. If using, add salt, sugar, and pepper.
```
Comment (bast) [inline homemade_tomato_sauce:14]: Here a comment on a particular line.

File: homemade_tomato_sauce [hunk 2]
```diff
@@ -2,8 +2,17 @@
 
 ## Ingredients
 
-1. 
-2. 
-3. 
+1. 1 kg Fresh tomatoes
+2. 2 spones Olive oil
```
Comment (bast) [inline homemade_tomato_sauce:6]:
```suggestion
2. 2 spoons Olive oil
```

File: homemade_tomato_sauce [hunk 3]
```diff
@@ -2,8 +2,17 @@
 
 ## Ingredients
 
-1. 
-2. 
-3. 
+1. 1 kg Fresh tomatoes
+2. 2 spoons Olive oil
```
Comment (EmadNB) [inline homemade_tomato_sauce:6]: I already tried this recipe and it's better to add more oil
```suggestion
2. 2.5 spoons Olive oil
```

File: homemade_tomato_sauce [hunk 4]
```diff
@@ -2,8 +2,17 @@
 
 ## Ingredients
 
-1. 
-2. 
-3. 
+1. 1 kg Fresh tomatoes
```
Comment (luisdavidpabonospina) [inline homemade_tomato_sauce:5]:
```suggestion
1. 5 kg Fresh tomatoes
```

**Pull request merged**